themes/classic/templates/errors/maintenance.tpl

```diff
@@ -17,9 +17,9 @@
     {block name='page_content_container'}
       <section id="content" class="page-content page-maintenance">
         {block name='page_content'}
-          {l s='We are currently updating our shop and will be back really soon.'}
-          <br>
-          {l s='Thanks for your patience.'}
+          {if isset($maintenance_text) && !empty($maintenance_text)}
+          {$maintenance_text nofilter}
+          {/if}
```

Comment:
`$maintenance_text` will always be set so you can remove check on isset and empty. It will be easier to maintain.